FindFirstAndLastElementPositionInSortedArray.java

```diff
@@ -0,0 +1,75 @@
+// Time Complexity : O(logn)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+
+// Approach: Use two different helper methods whis are binary search
+// with different start and end points.
+// For finding the first index, if we find nums[mid] >= target,
+// we move towards the left subarray by making end = mid - 1.
+// For finding the last index, if we find nums[mid] <= target,
+// we move towards right subarray by making start = mid + 1.
+
+class Solution {
```

Comment:
Try to write comments within code to explain what the code is doing for better understanding when you revisit the code in the future.